app/client/App.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import './app.css';
+import Table from './Table'
+
+export default class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { data: null };
+  }
+
+  componentDidMount() {
+    fetch('/api/getSchedule')
+      .then(res => res.json())
+      .then(res => {
+        this.setState({ data: res });
+      });
+  }
+
+  render() {
+
+    if (!this.state.data) {
+      return <div>Loading...</div>
+    }
+
+    return (
+      <div>
+        {this.state.data.map((day, idx) => <Table name={idx} data={day}></Table>)}
+      </div>
+    )
```

Comment:
Try to use something like that inside your return method, its gonna be easier to understand. 
```
<div> 
    {!this.state.data ? (
    "Loading...
    ) : (
    this.state.data.map(day, idx) => <Table name={idx} data={day}} />
</div>`
```